runtime/src/lib.rs

```diff
@@ -394,6 +394,7 @@ impl CreateRecoveryCallFactory<RuntimeOrigin, AccountId, BlockNumber> for Pallet
 }
 
 impl pallet_verified_recovery::Config for Runtime {
+	type LocId = u128;
```

Comment:
The `LocId` type could be defined one level above and shared between `pallet_logion_loc` and `pallet_verified_recovery`.